src/mnemonics/electrum-words.cpp

```diff
@@ -426,15 +423,43 @@ namespace crypto
 
         memwipe(w, sizeof(w));
       }
-
+      
+      std::tm genesis = {};
+      genesis.tm_year = 118;
+      genesis.tm_mon = 4;
+      genesis.tm_mday = 2;
+      
+      std::time_t genesis_time = std::mktime(&genesis);
+
+      int diff = (int) (std::difftime(seed_time, genesis_time) / (60*60*24));
+      int extra = 0;
+      while (diff > 1680) // Number of words in english.h
```

Comment:
1680 isn't the right number, either: there are 1626 seed words (i.e. a mnemonic seed is a base-1626 encoding of the 2^256 private spend key, because 1626^24 is slightly larger than 2^256).

Reply:
> If it's english, then it uses NWORDS ...

*All* the language uses 1626 (because 1626^24 is slightly larger than the values needed to encode every possible 2^256 private spend key).

But yes, it should just be using NWORDS.